modules/099-ceph-csi/crds/doc-ru-cephcsi.yaml

```diff
@@ -0,0 +1,119 @@
+spec:
+  versions:
+    - name: v1alpha1
+      schema: &schema
+        openAPIV3Schema:
+          type: object
```

Comment:
There should be only descriptions in a doc-ru file without `type`, `required`, and other properties.

Reply:
Unnecessary properties have been removed.